rules/linux/execution_perl_tty_shell.toml

```diff
@@ -11,19 +11,20 @@ interactive tty after obtaining initial access to a host.
 """
 from = "now-9m"
 index = ["auditbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
```

Comment:
can we revert this - #1730

Reply:
this _should_ be failing due to #1855 ...

Reply:
fix in #1880

Reply:
failed [test](https://github.com/elastic/detection-rules/runs/5684775206?check_suite_focus=true)

```
ValueError: cannot change "type" in locked rule: Interactive Terminal Spawned via Perl from query to eql
```

Reply:
How do I revert it?